content/home/marketing/learn-once-write-anywhere.md

```diff
@@ -1,8 +1,8 @@
 ---
-title: Learn Once, Write Anywhere
+title: Leer Eenmaal, Gebruik Het Overal
```

Comment:
```suggestion
title: Leer het Een Keer, Gebruik Het Overal
```

Reply:
Is there a particular reason that you didn't capitalize `het`?

Reply:
The English docs don't capitalize the article or prepositions.

Reply:
This is an issue elsewhere as well. At this point, it seems to make more sense to me to capitalize all words in Dutch. What do you guys think?

Reply:
I think the capitalization is a very American thing, not sure if we should follow this in the dutch translations